src/config/jwt.js

```diff
@@ -0,0 +1,33 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const ExtractJWT = passportJWT.ExtractJwt;
+const Strategy = passportJWT.Strategy;
+
+const { getUser } = require('../services/users-services');
+
+require("dotenv").config();
+const SECRET = process.env.SECRET;
+const params = {
+  secretOrKey: SECRET,
+  jwtFromRequest: ExtractJWT.fromAuthHeaderAsBearerToken(),
+};
+
+// const User = require("../models/users-schema");
+
+const setJWTStrategy = () => {
+  passport.use(
+    new Strategy(params, async (payload, done) => {
+      try {
+        const user = await getUser({ _id: payload.id });
+        if (!user) {
```

Comment:
```suggestion
        if (!user || !user.token) {
```
Upewnij się że token jest obecny

Reply:
Osoba ktora sie wyloguje chociaz ma ważny token (1d) to nadal nie moze miec dostepu.

Reply:
Ok, rozumiem. Poprawione.